### PR TITLE
SUIT-13274 Use translation module for non started reason and interactive progress

### PR DESCRIPTION
**Pull request by andriichumak**

### CODE REVIEW

File: test/utils/translateResults.test.js [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+		it('should transform markdown correctly', () => {
+			assert.strictEqual(
+				transformMarkdown(
+					'[link](url), ' +
+					'[email](mailto:sales@test.com), ' +
+					'[link](url){:target="_blank"}, ' +
+					'![preview](src), ' +
+					'**bold**'
+				),
+				'url, sales@test.com, url, src, bold',
```
Comment (andriichumak) [inline test/utils/translateResults.test.js]: `link (url), email (sales@test.com), link (url), preview (src), bold`.

Also, special case if link === url
`[url](url), [email](mailto:email), preview` => `url, email, preview`

File: lib/api/webSockets.js [hunk 2]
```diff
@@ -12,7 +12,8 @@ const SuitestError = require('../utils/SuitestError');
 const texts = require('../texts');
 const logger = require('../utils/logger');
 const {handleProgress} = require('../utils/interactiveProgressHanlder');
-const {getInfoErrorMessage, getNotStartedReasonMessage} = require('../utils/socketErrorMessages');
+const {getInfoErrorMessage} = require('../utils/socketErrorMessages');
+const {translateStartupError} = require('../utils/translateResults');
```
Comment (givehug) [inline lib/api/webSockets.js]: should be changed to translateNotStartedReason

**Comment by andriichumak** [comment]
On top we need to fix conflicts and add error handling (now translation module can throw.)

**Comment by andriichumak** [comment]
@givehug please update reference to `@suitest/translate` to `^2.0.1`

**Pull request merged**